lib/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -422,9 +422,16 @@ sub find_recent_by_artists
 {
     my ($self, $artist_ids) = @_;
 
-    my $search_url = sprintf("http://%s/ws/2/recording/?query=title:\"\"",
-                              DBDefs->LUCENE_SERVER);
-
+    if (DBDefs->SEARCH_ENGINE == 'LUCENE')
```

Comment:
s/==/eq/

In perl, numeric comparisons use `==` and string comparisons use `eq`.

This will always evaluate to true because it coerces both strings to `0` and then compares them numerically.